utility_scripts/deploy/deploy_hyperion.py

```diff
@@ -8,6 +9,8 @@
 
 recognised_beamlines = ["dev", "i03", "i04"]
 
+VERSION_PATTERN = re.compile(r"^v?(\d+)\.(\d+)\.(\d+)$")
```

Comment:
Could: A more comprehensive pattern is in `packaging.version.VERSION_PATTERN`, we could use that instead so we don't have to maintain our own?

Reply:
done